src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,23 @@
 import React from 'react';
+import { User } from '../../types/User';
+import './UserInfo.scss';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+interface UserInfoProps {
+  user: User | null;
```

Comment:
The user will never be null 
<img width="413" alt="image" src="https://github.com/mate-academy/react_static-list-of-posts/assets/99403198/cc8748fe-96f0-4a9a-98c8-3c9043099272">
```suggestion
  user: User;
```